addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3480,7 +3480,9 @@ const SnippetOptionWidget = Widget.extend({
                     obj = createPropertyProxy(this, '$target', $firstSubTarget);
                 }
 
+                this.options.wysiwyg.odooEditor.observerUnactive("computeWidgetState");
```

Comment:
- You should have added a comment to explain what this is about... but in fact it is not perfect: I forgot that `_computeWidgetState` is async. It probably is ok and we would not see any bug because of this... but this is technically not right to pause the observer for the duration of an async operation. Let's go back to what you did and pause the observer during the synchronous action that edit the DOM inside some `_computeWidgetState` implementation :+1: (still should be done in 15.0 (in this PR) rather than in your original PR :+1:)
- When adapting your commit message also fix this: 
![image](https://github.com/odoo/odoo/assets/10338094/101d68fc-676d-48b7-8006-ebc63ca7da77)

There should be a blank space after the title, please be careful. I cannot spot that error on github but it breaks the display of the commit message in common tools (like tig).

Reply:
@qsm-odoo , it seems that the problem has been resolved meanwhile by [this commit](https://github.com/odoo/odoo/commit/994fe756db8f1a45ef864cb7848644ea204539b7). Consequently, the runbot of [this PR](https://github.com/odoo/odoo/pull/108684) is green now :+1:

Reply:
So close this PR I guess? :+1: